pallets/delegation/src/mock.rs

```diff
@@ -176,24 +179,30 @@ pub mod runtime {
 	use sp_keystore::{testing::KeyStore, KeystoreExt};
 	use sp_runtime::{
 		testing::Header,
-		traits::{BlakeTwo256, IdentifyAccount, IdentityLookup},
-		MultiSigner,
+		traits::{BlakeTwo256, IdentifyAccount, IdentityLookup, Verify},
+		MultiSignature, MultiSigner,
 	};
 	use sp_std::sync::Arc;
 
+	use attestation::{mock::insert_attestation, AttestationDetails, ClaimHashOf};
 	use kilt_support::{
 		mock::{mock_origin, SubjectId},
 		signature::EqualVerify,
 	};
-	use runtime_common::constants::delegation::DELEGATION_DEPOSIT;
 
 	pub type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
 	pub type Block = frame_system::mocking::MockBlock<Test>;
 
-	type TestDelegationNodeId = runtime_common::Hash;
-	type TestDelegateSignature = (SubjectId, Vec<u8>);
-	type TestBalance = runtime_common::Balance;
-	type TestCtypeHash = runtime_common::Hash;
+	pub type Hash = sp_core::H256;
+	pub type Balance = u128;
+	pub type Signature = MultiSignature;
+	pub type AccountPublic = <Signature as Verify>::Signer;
+	pub type AccountId = <AccountPublic as IdentifyAccount>::AccountId;
+
+	pub const UNIT: Balance = 10u128.pow(15);
```

Comment:
What is this value used for?

Reply:
[1016572](https://github.com/KILTprotocol/mashnet-node/pull/316/commits/1016572af6dc9a4138bdeec705153437010a2e0b)